src/it/resources-noncompilable/com/google/checkstyle/test/chapter4formatting/rule451wheretobreak/InputNoWrappingAfterRecordName.java

```diff
@@ -0,0 +1,41 @@
+package com.google.checkstyle.test.chapter4formatting.rule451wheretobreak;
+
+/** Some javadoc. */
+public record InputNoWrappingAfterRecordName (String name) {
+  // violation above ''(' is preceded with whitespace'
```

Comment:
I don't see placing of this file to list of file to skip auto  formatting.
Is this a gap in formatter?